### PR TITLE
Rewrote all templates to use _ instead of -

### LINKED ISSUE #3612: rewrite all templates to use _ instead of -
we updated `{% block %}` defs to use _ rather than -, but there are a bunch of templates left that still mix those two.

See #3533

### PR DESCRIPTION
**Pull request by youriwims**
⚠️ Messed up branch 🤢

### CODE REVIEW

**Comment by youriwims** [comment]
@Pomax not sure what happened here because I definitely created this branch from master, but I see commits from #3637 😕

File: network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_card.html [hunk 1]
```diff
@@ -22,16 +22,11 @@
...
-      {% with tag=page.specific.tags.first %}
-        {% if tag %}
-          {% if root %}
-            {# If we have a "root" context variable, we know this card is generated on an index page (or index page subroute) #}
-            <a class="main-tag d-block mt-3" href="{% routablepageurl root.specific "entries_by_tag" tag.slug %}">{{ tag }}</a>
-          {% else %}
-            {# If we do not have a "root" context variable, this card is being generated outside of an index page, and so needs to use its parent as root #}
-            <a class="main-tag d-block mt-3" href="{% routablepageurl page.get_parent.specific "entries_by_tag" tag.slug %}">{{ tag }}</a>
-          {% endif %}
+      {% with category=page.specific.category.first %}
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_card.html]: this should purely be a refactor PR, let's not slip in code _changes_ =)

File: network-api/networkapi/wagtailpages/templatetags/blog_tags.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from django import template
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/templatetags/blog_tags.py]: Let's not add new files in this PR

File: source/sass/views/blog.scss [hunk 3]
```diff
@@ -17,9 +17,3 @@
     }
   }
 }
-
```
Comment (Pomax) [inline source/sass/views/blog.scss]: please revert this, too.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_card.html (hunk 1 above)
Comment (youriwims) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_card.html]: I didn't that's [the problem](https://github.com/mozilla/foundation.mozilla.org/pull/3649#issuecomment-531397383) 😭 Those changes are from a separate branch so, I'm not sure how this even happened.

**Comment by youriwims** [comment]
Superseded by #3652

**Pull request closed without merging**